src/Raven.Client/Documents/Session/IAdvancedSessionOperations.Patch.cs

```diff
@@ -32,8 +32,14 @@ public partial interface IAdvancedSessionOperations
         void Patch<T, TKey, TValue>(T entity, Expression<Func<T, IDictionary<TKey, TValue>>> path,
             Expression<Func<JavaScriptDictionary<TKey, TValue>, object>> dictionaryAdder);
 
-        void Patch<T, TKey, TValue>(string id, Expression<Func<T, IDictionary<TKey, TValue>>> path,
+        void Patch<T, TKey, TValue>(string id, Expression<Func<T, IDictionary<TKey, TValue>>> patch,
```

Comment:
```suggestion
        void Patch<T, TKey, TValue>(string id, Expression<Func<T, IDictionary<TKey, TValue>>> path,
```

This is the path to patch, not a patch itself

Reply:
done